tests/webproxy.nix

```diff
@@ -47,9 +81,10 @@ import ./make-test-python.nix ({ pkgs, ... }:
     with subtest("varnishncsa should log requests"):
         webproxy.wait_until_succeeds(f"{curl} && grep -q 'GET {url} HTTP/' /var/log/varnish.log")
 
-    with subtest("changing config and reloading should activate new config"):
-        webproxy.execute('ln -sf /etc/local/varnish/newconfig.vcl /etc/current-config/varnish.vcl')
-        webproxy.succeed('systemctl reload varnish')
-        webproxy.succeed('varnishadm -n /var/spool/varnish/webproxy vcl.list | grep active | grep -q newconfig')
+    with subtest("varnish pid should be the same across small configuration changes"):
```

Comment:
Getting there ... :)

I'd appreciate if you could double check that the configs (as visible in varnishadm) are as expected and are changed in the reload.